web/src/components/ChatConversation.tsx

```diff
@@ -41,7 +41,7 @@ const ChatConversation = ({ messages, hasError, className }: ChatConversationPro
   const [showTypingIndicator, setShowTypingIndicator] = useState(false)
   const messagesEndRef = useRef<HTMLDivElement>(null)
   const showIndicatorOfTyping =
```

Comment:
:upside_down_face: 
```suggestion
  const showTypingIndicator =
```

Reply:
we have the same naming for the state above :)

Reply:
Oh sorry, I missed that. I think that we should think of a more explicit naming. How about something like `shouldShowTypingIndicator` and `typingIndicatorVisible`?